mlir/lib/Dialect/Bufferization/Transforms/Bufferize.cpp

```diff
@@ -75,7 +75,7 @@ BufferizeTypeConverter::BufferizeTypeConverter() {
       if (!rankedDestType)
         return nullptr;
       FailureOr<Value> replacement =
-          castOrReallocMemRefValue(builder, inputs[0], rankedDestType);
+          castOrReallocMemRefValue(builder, inputs[0], rankedDestType, {});
```

Comment:
Can you pass a `BufferizationOptions` object with `bufferAlignment = 0` here? Otherwise it will fall back to the default value of `64`. This should reduce the number of changed test cases.

Reply:
done